src/containers/Preview/PlaylistPreview/style.scss

```diff
@@ -607,3 +607,46 @@ img {
     }
   }
 }
+
+@media(max-width:991px){
+  .flex-container.previews{
+    flex-wrap: wrap;
+    
+    @media(max-width:767px){
+      flex-wrap: wrap-reverse;
+    }
+
+    .right-sidegolf-info{
+      width:100%
```

Comment:
Do not forget semi colon on style sheet
It will be broken on iOS devices

Reply:
added